app/components/provider-links/component.js

```diff
@@ -0,0 +1,30 @@
+import { inject as service } from '@ember/service';
+import Component from '@ember/component';
+import { task } from 'ember-concurrency';
+
+export default Component.extend({
+    store: service(),
+    theme: service(),
+
+    tagName: 'ul',
+    classNames: ['fa-ul'],
+
+    init() {
+        this._super(...arguments);
+        this.iconMap = {
+            preprint: 'fa-graduation-cap',
+            paper: 'fa-file-text-o',
+            thesis: 'fa-book',
+        };
+        this.pendingCount = null;
+        this.get('fetchData').perform();
+    },
+
+    fetchData: task(function* () {
+        const provider = yield this.get('theme').loadProvider(this.get('provider.id'));
+        const results = yield this.get('store').queryHasMany(provider, 'preprints', {
+            'meta[reviews_state_counts]': true,
```

Comment:
The counts are [available on the provider](https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/models/preprint-provider.js#L25), if you use `?related_counts=true`.

That only works when fetching one provider at a time, which would be fine, but if we wanted we could make a [two-line change to the backend](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/actions/serializers.py#L38) (would have sworn it already worked in the list view, alas) and get all the counts in that one request in `load-providers`.

Reply:
Ahhhhh I was looking for that and I never found it...